starter-code/index.js

```diff
@@ -1,17 +1,56 @@
-class SortedList {
-  constructor() {}
-
-  add(item) {}
-
-  get(pos) {}
-
-  max() {}
-
-  min() {}
-
-  sum() {}
-
-  avg() {}
-}
-
-module.exports = SortedList;
+class SortedList {
+  constructor(items, length) {
+    this.items = [];
+    this.length = this.items.length;
+  }
+
+  add(item) {
+    this.items.push(item);
+    this.length = this.items.length;
+    this.items.sort((a, b) => a - b);
+  }
+
+  get(pos) {
+    if (pos > this.length) {
+      throw new Error("OutOfBounds");
+    } else {

```

Comment:
The else statement is not needed here. You can just write: return this.items[pos]